tests/integration/data/legacy/case-08.md

```diff
@@ -0,0 +1 @@
+# Case 8 Description
```

Comment:
Search with sorting?

Reply:
As for the other comment, I just left placeholders when there were no existing test description. I'll add the appropriate description.